tf-psa-crypto/tests/suites/test_suite_psa_crypto.function

```diff
@@ -10505,6 +10534,11 @@ void iop_export_public_key(
                                                 &output_len);
     TEST_EQUAL(status, PSA_ERROR_BAD_STATE);
 
+    PSA_ASSERT(psa_export_public_key_iop_abort(&export_key_operation));
+
+    num_ops = psa_export_public_key_iop_get_num_ops(&export_key_operation);
+    TEST_EQUAL(num_ops, 0);
+
```

Comment:
What about the case of calling `psa_export_public_key_iop_xxx` on a public key object? Is this in scope for this pull request (I guess so since https://github.com/Mbed-TLS/mbedtls/issues/9649 doesn't mention a limitation)?

Reply:
It is in scope. Added a test for ti as well.